### PR TITLE
ports/mimxrt: Use hardware byteswap support.

### PR DESCRIPTION
**Pull request by kwagyeman**
Broken out into a PR as requested from https://github.com/openmv/openmv/pull/2123

### CODE REVIEW

File: src/omv/boards/OPENMV_RT1060/omv_boardconfig.h [hunk 1]
```diff
@@ -211,4 +211,6 @@
 #define OMV_CSI_POWER_PIN               (&omv_pin_DCMI_POWER)
 #define OMV_CSI_FSYNC_PIN               (&omv_pin_DCMI_FSYNC)
 
+#define OMV_CSI_HAS_BSWAP               (1)
```
Comment (iabdalkader) [inline src/omv/boards/OPENMV_RT1060/omv_boardconfig.h:214]:
```suggestion
#define OMV_CSI_HW_SWAP_ENABLE          (1)
```
Can you please add this right after `OMV_CSI_XCLK_FREQUENCY` ? Options first, then pins.

File: src/omv/common/sensor_utils.c [hunk 2]
```diff
@@ -1307,13 +1307,17 @@ __weak int sensor_copy_line(void *dma, uint8_t *src, uint8_t *dst) {
             #if OMV_CSI_DMA_MEMCPY_ENABLE
             sensor_dma_memcpy(dma, dst16, src16, sizeof(uint16_t), sensor.transpose);
             #else
+            #if OMV_CSI_HAS_BSWAP
+            if (0) {
+            #else
```
Comment (iabdalkader) [inline src/omv/common/sensor_utils.c:1312]:
```suggestion
            if (0) {
            #if OMV_CSI_HAS_BSWAP
            } else
```

**Review by iabdalkader — CHANGES_REQUESTED** [review]

**Comment by iabdalkader** [comment]
Otherwise, looks good.

File: src/omv/common/sensor_utils.c (hunk 2 above)
Comment (kwagyeman) [reply src/omv/common/sensor_utils.c]: Done

File: src/omv/boards/OPENMV_RT1060/omv_boardconfig.h (hunk 1 above)
Comment (kwagyeman) [reply src/omv/boards/OPENMV_RT1060/omv_boardconfig.h]: Done

Comment (iabdalkader) [reply src/omv/boards/OPENMV_RT1060/omv_boardconfig.h]: Please use this option name `OMV_CSI_HW_SWAP_ENABLE`

Comment (kwagyeman) [reply src/omv/boards/OPENMV_RT1060/omv_boardconfig.h]: Done

**Review by iabdalkader — APPROVED** [review]

**Pull request merged**